script/DeployVault.sol

```diff
@@ -0,0 +1,250 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import {
+  IEarnStrategyRegistry, EarnStrategyRegistry
+} from "@balmy/earn-core/strategy-registry/EarnStrategyRegistry.sol";
+import { FirewalledEarnVault } from "@balmy/earn-core/vault/FirewalledEarnVault.sol";
+import { EarnNFTDescriptor } from "@balmy/earn-core/nft-descriptor/EarnNFTDescriptor.sol";
+import { FirewalledEarnVaultCompanion, IPermit2 } from "src/companion/FirewalledEarnVaultCompanion.sol";
+import { ICreationValidationManagerCore } from "src/interfaces/ICreationValidationManager.sol";
+
+import { GlobalEarnRegistry } from "src/global-registry/GlobalEarnRegistry.sol";
+import { FeeManager, Fees } from "src/strategies/layers/fees/external/FeeManager.sol";
+import { GuardianManager } from "src/strategies/layers/guardian/external/GuardianManager.sol";
+import { LiquidityMiningManager } from "src/strategies/layers/liquidity-mining/external/LiquidityMiningManager.sol";
+import { DelayedWithdrawalManager } from "src/delayed-withdrawal-manager/DelayedWithdrawalManager.sol";
+import { TOSManager } from "src/strategies/layers/creation-validation/external/TOSManager.sol";
+import { GlobalValidationManagersRegistry } from
+  "src/strategies/layers/creation-validation/external/GlobalValidationManagersRegistry.sol";
+import { SignatureBasedWhitelistManager } from
+  "src/strategies/layers/creation-validation/external/SignatureBasedWhitelistManager.sol";
+import { ExternalFirewall } from "@forta/firewall/ExternalFirewall.sol";
+import {
+  FirewallAccess,
+  FIREWALL_ADMIN_ROLE,
+  PROTOCOL_ADMIN_ROLE,
+  ATTESTER_MANAGER_ROLE,
+  CHECKPOINT_EXECUTOR_ROLE,
+  TRUSTED_ATTESTER_ROLE
+} from "@forta/firewall/FirewallAccess.sol";
+import { ISecurityValidator } from "@forta/firewall/SecurityValidator.sol";
+import { FirewallRouter } from "@forta/firewall/FirewallRouter.sol";
+
+import { ICheckpointHook } from "@forta/firewall/interfaces/ICheckpointHook.sol";
+import { Checkpoint, Activation } from "@forta/firewall/interfaces/Checkpoint.sol";
+
+import { BaseDeploy } from "./BaseDeploy.sol";
+import { console2 } from "forge-std/console2.sol";
+
+contract DeployVault is BaseDeploy {
+  function run() external {
```

Comment:
I think we can make this more readable and re-usable for future updates if we split this script into different files. Like one for the strategy registry, another for the vault, another for the different managers, etc

Is there a way to do that?